### PR TITLE
[Port] l10n_ch_hr_payroll to v9

### PR DESCRIPTION
**Pull request by open-net-sarl**
Module ported to v8.
Commision and reimbursement functionalities moved to hr_payroll_extended.

### CODE REVIEW

**Comment by eLBati** [comment]
Thanks @open-net-sarl , I think you meant "to v9"

**Comment by open-net-sarl** [comment]
Sorry @eLBati, "to v9" of course.

**Comment by cyp-opennet** [comment]
:+1:

**Comment by dcoopennet** [comment]
Please review this module ported to v9.

File: l10n_ch_hr_payroll/__init__.py [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
-
-from . import models
+# -*- coding: utf-8 -*-
+#
+#  File: __init__.py
+#  Module: hr_payroll_extended
```
Comment (gurneyalex) [inline l10n_ch_hr_payroll/__init__.py]: this does not match the module name.

**Comment by open-net-sarl** [comment]
Hello,
Anyone to review this PR ?
Thanks

**Comment by open-net-sarl** [comment]
@yvaucher @eLBati @gurneyalex 

Hi,

Could you please review this module ?

Thanks

File: l10n_ch_hr_payroll/__init__.py [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (leemannd) [inline l10n_ch_hr_payroll/__init__.py]: Short header here

File: l10n_ch_hr_payroll/models/__init__.py [hunk 3]
```diff
@@ -1,30 +1,30 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (leemannd) [inline l10n_ch_hr_payroll/models/__init__.py]: Short header here

File: l10n_ch_hr_payroll/models/hr_contract.py [hunk 4]
```diff
@@ -1,63 +1,40 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (leemannd) [inline l10n_ch_hr_payroll/models/hr_contract.py]: Short header here

File: l10n_ch_hr_payroll/models/hr_employee.py [hunk 5]
```diff
@@ -1,38 +1,39 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (leemannd) [inline l10n_ch_hr_payroll/models/hr_employee.py]: Short header here

**Pull request merged**